swap_meet/clothing.py

```diff
@@ -1,2 +1,18 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+    '''
+    Represents Clothing Item with a fabric attribulte.
+    Inherites from Items.
+    '''
+    
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        super().__init__(id, condition)
+        self.fabric = fabric
+    
+    def __str__(self):
+        return f"An object of type Clothing with id {self.id}. It is made from {self.fabric} fabric."  
+    
+    def get_category(self):
+        return "Clothing"
```

Comment:
For our children classes of `Item`,  we have a `get_category` method that essentially returns the name of the class. Though we are returning the different names (Clothing, Decor, Electronic) we could actually move this functionality into the parent class, `Item`. This is how it would look: 
```python
class Item:
...
def  get_category(self):
	return  self.__class__.__name__
```
`self` refers to the class instance that called it. Meaning if we have the following: 
```python
shirt = Clothing()

shirt.get_category() # returns "Clothing"
```
`self` in this case refers to `shirt`. The class of clothing is `Clothing`  and since `Clothing` is inheriting from the `Item` class it has access to the `get_category` method. This method is inherently being passed `shirt` and then accesses the `__class__` attribute and then the `__name__` attribute which gives us `Clothing`. Doing this will allow to make our code more DRY since we do this for `Clothing`, `Decor`, and `Electronic`.